manifests/opendatahub/base/deployments/ml-pipeline-visualizationserver.yaml

```diff
@@ -19,7 +19,7 @@ spec:
         application-crd-id: kubeflow-pipelines
     spec:
       containers:
-        - image: visualization-server
+        - image: visualization-server:1.3.1
```

Comment:
I think this image is replaced via the kustomization, so specifying a tag here would just be overwritten, right?  I think we should update the `manifests/opendatahub/base/kustomization.yaml` with this image instead